library/lib_handler.php

```diff
@@ -1,15 +1,15 @@
 <?php
 require("utilities/functions.php");
-require("database/connection.php");
+require("../../database/connection.php");
 require("lib_main.php");
 require("lib_sql_que.php");
 require("log_debug.php");
-require("model/model.php");
-require('model/blog.php');
-require('model/article.php');
-require('model/blog-category.php');
-require('model/blog_post_categories.php');
-require('model/blog_post_comment.php');
-require('model/user.php');
+require("../models/model.php");
+require('../models/blog.php');
+require('../models/article.php');
+require('../models/blog-category.php');
+require('../models/blog_post_categories.php');
+require('../models/blog_post_comment.php');
+require('../models/user.php');
 session_start();
 ?>
```

Comment:
alisin mo na to

Reply:
yes kuys bali sa last commit ko tinanggal ko na kuys, thank you